node/src/BaseClient.ts

```diff
@@ -1079,8 +1086,8 @@ export class BaseClient {
      * ```
      */
     public async set(
-        key: string | Uint8Array,
-        value: string | Uint8Array,
+        key: GlideString | Uint8Array,
```

Comment:
```suggestion
        key: GlideString,
        value: GlideString,
```
Remove the Uint8Array type.  This is from @eifrah-aws prototype code and can be removed.

Reply:
Original change: https://github.com/valkey-io/valkey-glide/commit/7df249889b7915399a23e07353d25854b348e0bb